Routes/ArticlePage.jsx

```diff
@@ -0,0 +1,31 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { getArticleById } from "../src/components/utils/utils";
+import ArticlePageCard from "../src/components/ArticlePageCard";
+
+const ArticlePage = () => {
+  const { article_id } = useParams();
```

Comment:
Please include an error state here to display a message in case anything goes wrong.